common/achievements/defeat-evil-x.ts

```diff
@@ -0,0 +1,19 @@
+import {achievement} from './defaults'
+import {Achievement} from './types'
+
+const DefeatEvilX: Achievement = {
+	...achievement,
+	numericId: 6,
+	id: 'defeat_evil_x',
+	name: 'Evil X-terminated',
+	description: 'Defeat Evil X',
+	steps: 1,
+	onGameEnd(game, playerEntity, component, outcome) {
+		if (!game.state.isBossGame) return
```

Comment:
Can you rename this too `isEvilXBossGame` so when I go back to redo how this works I don't screw it up?

Reply:
Done!